test/files/neg/i15552.scala

```diff
@@ -0,0 +1,34 @@
+
+class C {
+  val ctx: C = new C()
+  def f() = g(42)(using ctx)  // error: no g
+}
+
+class Innocent {
+  val using = 42
+  def g(i: Int) = i
+  def f() = g(using)
```

Comment:
`g(using using)` works too 😀

Reply:
I wish I had thought of that one. Maybe I'll add another test class `MaliciousPuns`.